package.json

```diff
@@ -16,6 +16,7 @@
     "**/*.{js,jsx,ts,tsx}": "prettier --write"
   },
   "dependencies": {
+    "@headlessui/react": "^1.7.10",
```

Comment:
I don't have a lot of experience with transitions in react but let's try to use [react-transition-group](https://github.com/reactjs/react-transition-group/tree/v1-stable) which seems to be the package [suggested by the React team](https://reactjs.org/docs/animation.html), instead of adding a new ui library just so we can use their `Transition` component.

Reply:
Addressed in b5210a1c3c0612450de52ddea543f2984248d6bd by using plain tailwind.